### DISCUSSION TITLE
Issue with displaying localhost web page on Lively Wallpaper

### DISCUSSION THREAD

**Discussion by fantasyx10** [rocksdanister/lively#1696] [Q&A]
I am writing to seek assistance with an issue that I am experiencing with Lively Wallpaper. Specifically, I am having trouble getting Lively Wallpaper to display a localhost web page properly.

I have followed the steps outlined in the documentation, including ensuring that my web server is running on localhost, setting the "Webpage" option in Lively Wallpaper to "Local URL", and inputting the appropriate URL with the desired page path. However, despite these efforts, the web page is not displaying properly on Lively Wallpaper.

I have also checked my firewall settings to ensure that Lively Wallpaper is allowed to access the local host. Despite this, the issue persists.

Could you please advise on any other troubleshooting steps that I could take to resolve this issue? I appreciate any assistance you can provide.

**Comment by rocksdanister** [comment]
>I have followed the steps outlined in the documentation, including ensuring that my web server is running on localhost, setting the "Webpage" option in Lively Wallpaper to "Local URL", and inputting the appropriate URL with the desired page path. However, despite these efforts, the web page is not displaying properly on Lively Wallpaper.

I don't remember writing this, can you share the documentation link?

As for the issue itself, clearly layout the steps you followed to reproduce the issue along with screenshots if it helps:
1. Start localhost server using..
2. Open Add Wallpaper(+) dialog.
3. Enter "http://localhost:8000/"...
